kafka_consumer/tox.ini

```
@@ -2,7 +2,8 @@
 minversion = 2.0
 basepython = py37
 envlist =
-    py{27,37}-{0.10.2,0.11,1.1,2.3,latest}-{kafka,zk}
+    py27-latest-{kafka,zk}
```

Caution: the PR description reads `py27-{2.3}-{kafka,zk}`…?

```suggestion
    py27-2.3-{kafka,zk}
```

---

Oh I got it, the version of Kafka is actually irrelevant here since we're only aimaing at testing Python 2 x {zk, kafka}